macros/src/openapi/derive.rs

```diff
@@ -221,6 +221,45 @@ fn handle_fields(type_attrs: &[Attribute], fields: &mut Fields) -> Block {
     block
 }
 
+fn handle_fields_required(type_attrs: &[Attribute], fields: &Fields) -> Expr {
+    let reqf_v: Punctuated<Expr, Token![,]> = fields
+        .iter()
+        .map(|f| {
+            Pair::Punctuated(
+                q!(
+                    Vars {
+                        name_str: field_name(type_attrs, &*f),
+                        Type: &f.ty
+                    },
+                    {
+                        {
+                            if !<Type as rweb::openapi::Entity>::describe()
+                                .nullable
+                                .unwrap_or(false)
+                            {
+                                Some(rweb::rt::Cow::Borrowed(name_str))
+                            } else {
+                                None
+                            }
+                        }
+                    }
+                )
+                .parse(),
+                Default::default(),
+            )
+        })
+        .collect();
+
+    if reqf_v.is_empty() {
+        q!({ vec![] })
+    } else {
+        q!(Vars { reqf_v }, {
+            vec![reqf_v].into_iter().flatten().collect()
```

Comment:
Doesn't `vec![reqf_v]` work?

Reply:
Here `reqf_v` are elements typed `Option<Str>` (see [l240-242](https://github.com/kdy1/rweb/pull/59/files#diff-652ac060d5f66badbfcf5b3305d2cfd8b38157138a92f93e4b0f38090da1a135R240-R242)). So we have `Vec<Option<Str>>` and want `Vec<Str>`.